RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.cc

```diff
@@ -0,0 +1,90 @@
+#include "RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.h"
+#include "DataFormats/ForwardDetId/interface/HGCEEDetId.h"
+#include "DataFormats/ForwardDetId/interface/HGCHEDetId.h"
+#include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "CommonTools/Utils/interface/StringToEnumValue.h"
+
+HGCalRecHitWorkerSimple::HGCalRecHitWorkerSimple(const edm::ParameterSet&ps) :
+        HGCalRecHitWorkerBaseClass(ps)
+{
+        rechitMaker_ = new HGCalRecHitSimpleAlgo();
+	// HGCee constants 
+	HGCEEmipInKeV_ =  ps.getParameter<double>("HGCEEmipInKeV");
+	HGCEElsbInMIP_ =  ps.getParameter<double>("HGCEElsbInMIP");
+	HGCEEmip2noise_ = ps.getParameter<double>("HGCEEmip2noise");
+	hgceeADCtoGeV_ = HGCEEmipInKeV_ * HGCEElsbInMIP_/1000000; 
```

Comment:
please introduce a constant for the 1000000; better use *1.0E-6; it may be easier to read than to count zeroes.